server/plugins/connect-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/connect/obmysql/OBMySQLConnectionExtension.java

```diff
@@ -117,9 +117,8 @@ protected String getJdbcUrlParameters(Map<String, String> jdbcUrlParams) {
     }
 
     protected Map<String, String> appendDefaultJdbcUrlParameters(Map<String, String> jdbcUrlParams) {
-        if (!jdbcUrlParams.containsKey("enableFullLinkTrace")) {
-            jdbcUrlParams.put("enableFullLinkTrace", "true");
-        }
+        // there is a bug of oceanbase-client 2.4.7.1, setting this to true may cause OOM exception
```

Comment:
will it leads to full link trace not works? when wil oceanbase-client fix this OOM issue

Reply:
No, this setting only affects the stage of jdbc. The remaining stages of obproxy and observer can still be displayed normally.

Reply:
They will fix this in 2.4.8.